src/components/DopplerPlus/ExclusiveForm.js

```diff
@@ -184,6 +219,32 @@ const ExclusiveForm = ({ dependencies: { dopplerLegacyClient, appSessionRef } })
                       </fieldset>
                     </FieldItem>
                   </FieldGroup>
+                  <div className="dp-wrapper-volume-options" style={{ margin: '30px 0' }}>
```

Comment:
Why not create a styled component for this `div`? To avoid using inline style, because this feature will be here forever. 😄

Reply:
because its only for this element, but i can create

Reply:
changed